network/datadog_checks/network/network.py

```diff
@@ -466,13 +467,16 @@ def _get_net_proc_base_location(proc_location):
             net_proc_base_location = proc_location
         return net_proc_base_location
 
-    def _add_conntrack_stats_metrics(self, conntrack_path, tags):
+    def _add_conntrack_stats_metrics(self, conntrack_path, disable_sudo_conntrack, tags):
         """
         Parse the output of conntrack -S
         Add the parsed metrics
         """
         try:
-            output, _, _ = get_subprocess_output(["sudo", conntrack_path, "-S"], self.log)
+            if disable_sudo_conntrack is True:
+                output, _, _ = get_subprocess_output([conntrack_path, "-S"], self.log)
+            else:
+                output, _, _ = get_subprocess_output(["sudo", conntrack_path, "-S"], self.log)
```

Comment:
I'd write it a bit differently:
```
cmd = [conntrack_path, "-S"]
if use_sudo:
    cmd.insert(0, "sudo")
output, _, _ = get_subprocess_output(cmd, self.log)
```

And then pass `not disable_sudo_contrack` as argument instead.